code/game/gamemodes/miniantags/morph/spells/pass_airlock.dm

```diff
@@ -0,0 +1,50 @@
+// TODO refactor when spell code is component based instead of OO based
```

Comment:
Component based? :eyes:

Reply:
Not the components as in DM but more like described here https://en.wikipedia.org/wiki/Component-based_software_engineering 
The targeting on it's own can be a datum or so which you can just put on your spell at will. Which will reduce the OO madness where you can't make a morph AOE spell and a morph click based spell
Soon:tm:

Reply:
Oh ok I was thinking datum components, and I got excited for a minute :smile: Still, sounds like a good idea.